.gitignore

```diff
@@ -0,0 +1 @@
+/phonebook
```

Comment:
Why is this in the gitignore? ... Some of the files that need to be ignored are the __pycache__